org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/handlers/CodeActionHandler.java

```diff
@@ -53,33 +56,134 @@
 public class CodeActionHandler {
 
 	public static final String COMMAND_ID_APPLY_EDIT = "java.apply.workspaceEdit";
+	private static final String CODE_ACTION_KIND_QUICK_ASSIST = "quickassist";
```

Comment:
Create the new kind `QUICK_ASSIST` in `JavaCodeActionKind`.

Reply:
ok